datafusion-cli/src/exec.rs

```diff
@@ -388,15 +388,7 @@ mod tests {
         // Ensure that local files are also registered
         let sql =
             format!("CREATE EXTERNAL TABLE test STORED AS PARQUET LOCATION '{location}'");
-        let err = create_external_table_test(location, &sql)
```

Comment:
This no longer generates an error as we don't require that the path exists

Reply:
I swear there is a ticket that tracks this, but I couldn't find it. Thus I filed https://github.com/apache/arrow-datafusion/issues/8277 and updated this PR to close it